scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -3,56 +3,101 @@ package org.oppia.android.scripts.coverage
 import org.oppia.android.scripts.common.BazelClient
 import org.oppia.android.scripts.common.CommandExecutor
 import org.oppia.android.scripts.common.CommandExecutorImpl
+import org.oppia.android.scripts.common.ProtoStringEncoder.Companion.toCompressedBase64
 import org.oppia.android.scripts.common.ScriptBackgroundCoroutineDispatcher
 import org.oppia.android.scripts.proto.Coverage
+import org.oppia.android.scripts.proto.CoverageDetails
+import org.oppia.android.scripts.proto.CoverageExemption
+import org.oppia.android.scripts.proto.CoverageFailure
 import org.oppia.android.scripts.proto.CoverageReport
+import org.oppia.android.scripts.proto.CoverageReportContainer
 import org.oppia.android.scripts.proto.CoveredLine
 import org.oppia.android.scripts.proto.TestFileExemptions
 import java.io.File
 import java.util.concurrent.TimeUnit
 
+/** ANSI escape codes for colors. */
+/** Green text. */
+const val GREEN = "\u001B[32m"
+/** Red text. */
+const val RED = "\u001B[31m"
+/** Default text. */
+const val RESET = "\u001B[0m"
+/** Bold text. */
+const val BOLD = "\u001B[1m"
```

Comment:
@BenHenning, the color codes are being added to enhance user experience and provide visual cues. 

**FAILED**
![image](https://github.com/user-attachments/assets/4f7ba332-faab-491e-abea-39148618735e)

**PASSED**
![image](https://github.com/user-attachments/assets/991fed48-2866-4b46-b067-45fce1a25e38)

Their inclusion is subject to further discussion. If approved, they will be moved to a more appropriate location; if not, they will be removed.

Reply:
I'm totally fine with including TTY ANSI control codes here for colorizing--it's a nice touch. I don't have a strong opinion on centralizing the color locations, though I have a PR that I hope to revive one day that introduces a console class for all scripts which includes colorizing support (I suspect that would be the eventual place that these escape codes would live in the long-term).

Reply:
Thanks for confirming. I’ll keep the color codes here until a centralized place is decided in the future.